### ISSUE TITLE
http://swelljoe.com/ is down too...

### ISSUE THREAD

**Issue by LeChatNoir666** [swelljoe/swelljoe.com#1]
Just wanted to say thank you for fixing doxfer which helped me :)
Other guides and wikis of Webmin are Quiet outdated...

Oh and to report you that your http://swelljoe.com/ is down too...
Same server? :)

Again thank you and keep healthy!

**Comment by swelljoe** [comment]
Yes, same server, but swelljoe.com is down for different reasons and has been for ages. I don't even remember why, just haven't had time to work on it. It's nothing important.
